ratatui-widgets/src/block.rs

```diff
@@ -530,6 +534,14 @@ impl<'a> Block<'a> {
         self
     }
 
+    /// Sets the block's [`MergeStyle`] for overlapping characters. Setting it to `None`
+    /// never merges characters.
+    #[must_use = "method moves the value of self and returns the modified value"]
+    pub const fn merge_style(mut self, merge_style: Option<MergeStyle>) -> Self {
```

Comment:
This should talk about how this is related to the border and perhaps be named merge_borders or something similar. It definitely should not have style in the name.